ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/graphql/internal/pojoadapter/BlockAdapterBase.java

```diff
@@ -153,13 +151,7 @@
     @SuppressWarnings("unchecked")
     final List<List<Bytes32>> topics = (List<List<Bytes32>>) filter.get("topics");
 
-    final List<List<LogTopic>> transformedTopics = new ArrayList<>();
```

Comment:
Again, restoring this longer stanza in the name of type safety is OK for this PR.

Reply:
done